.github/workflows/ci.yml

```diff
@@ -475,7 +479,63 @@ jobs:
           GOPROXY: direct
         run: |
           make test
-        working-directory: src/github.com/containerd/containerd
+
+      - uses: actions/checkout@v2
+        with:
+          repository: ukontainer/runu
+          path: src/github.com/ukontainer/runu
+
+      - run: script/setup/install-dev-tools
+
+      - name: Install runtime
+        shell: bash
+        run: |
+          SHIM_COMMIT=v0.8.4
+          cd /tmp
+          GO111MODULE=on go get -v github.com/ukontainer/runu@"${SHIM_COMMIT}"
+          bindir="${{ github.workspace }}/bin/"
+          cd ${{ github.workspace }}/src/github.com/ukontainer/runu
+          git fetch --tags origin "${SHIM_COMMIT}"
+          git checkout "${SHIM_COMMIT}"
+          GO111MODULE=on go build -o "${bindir}/containerd-shim-darwin-v1" ./cmd/containerd-shim-darwin-v1
+
+      - name: Binaries
+        shell: bash
+        run: |
+          set -o xtrace
+          make binaries
+
+      - name: Install containerd
+        run: |
+          make binaries
+          sudo make install
+
+      - name: Integration 1
+        env:
+          TEST_RUNTIME: "io.containerd.darwin.v1"
```

Comment:
How about using ukcontainer or runu's namespace instead of `io.containerd`? For example firecracker-containerd is under `aws.firecracker`.

Reply:
https://github.com/containerd/containerd/pull/4526#discussion_r482694602

I received a similar comment before but didn't fix it; I'll change this to 'io.containerd.runu.v1' based on the previous comment.